content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -1541,6 +1541,48 @@ There's still a lot more to be done here: integrating the focus system with a [g
 
 Special thanks to `@mcwcampbell`, the lead author of AccessKit, for reviewing our integration and working with us to reduce the number of dependencies upstream, [substantially improving both compile times and final executable size](https://github.com/bevyengine/bevy/pull/6874#issuecomment-1440978453). This is [still a serious challenge on Linux](https://github.com/bevyengine/bevy/pull/6874#issuecomment-1432144117), and so the `accesskit_unix` feature flag is [disabled by default for now](https://github.com/bevyengine/bevy/pull/6874#issuecomment-1433896811).
 
+## Revamped Bloom
+<div class="release-feature-authors">authors: @StarLederer, @JMS55</div>
+
+Bloom has undergone some major changes and now looks better, is easier to control, and has fewer visual artifacts.
+In combination with the new tonemapping options, bloom has been much improved since the previous release!
+
+1. In Bevy 0.9, bloom looked like this.
+2. Switching the tonemapper to something like `AcesFitted` is already a big improvement.
+3. in Bevy 0.10, bloom now looks like this. It's much more controlled and less overbearing.
+4. To make the bloom stronger, rather than raise the `BloomSettings` intensity,
+let's double the `emissive` value of each cube.
+5. Finally, if you want more extreme bloom similar to the old algorithm, you can change
+`BloomSettings::composite_mode` from `BloomCompositeMode::EnergyConserving` to `BloomCompositeMode::Additive`.
+6. Explore the new bloom settings in an interactive playground using the new `bloom_3d` (and `bloom_2d`) examples.
+
+<div style="display: grid; grid-template-columns: repeat(2, 1fr); grid-gap: 16px;">
```

Comment:
```suggestion
<div style="display: grid; grid-template-columns: repeat(auto-fit, minmax(400px, 1fr)); grid-gap: 16px;">
```
This will use 2 column on screens wider than `850px` and 1 column on screens smaller than that.